.gitignore

```diff
@@ -39,3 +40,4 @@ callgrind.out.*
 tests/autotests/autotests
 .ycm_extra_conf.py*
 *.stash
+
```

Comment:
hmmm now that's odd. You are correct.... Mac editors are crap... every single one of them.